spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -472,3 +471,123 @@ definitions:
+            units: months
+            desc: Month of the year (UTC) (ranges from to 1 to 12)
+        - day:
+            type: u8
+            units: days
+            desc: Day of the month (UTC) (ranges from 1 to 31)
+        - mins:
+            type: u8
+            units: minutes
+            desc: Minute of the hour (UTC) (ranges from 0 to 59)
+        - sec:
+            type: u8
+            units: s
+            desc: |
+              Seconds of the minute (UTC) (ranges from 0 to 60). The upper range
+              includes an the possibility of an extra leap second correction.
+        - ns:
+            type: s32
+            units: ns
+            desc: Nanosecond residual of UTC sec (ranges from -1e9 to 1e9)
+        - lat:
+            type: double
+            units: deg
+            desc: Latitude
+        - lon:
+            type: double
+            units: deg
+            desc: Longitude
+        - height:
+            type: s32
+            units: mm
+            desc: Height above WGS84 ellipsoid
+        - height_msl:
+            type: s32
+            units: mm
+            desc: Height above mean sea level (not implemented)
+        - h_accuracy:
+            type: double
+            units: mm
+            desc: |
+              Horizontal position accuracy estimate (not
+              implemented). Defaults to 0.
+        - v_accuracy:
+            type: u16
+            units: mm
+            desc: |
+              Vertical position accuracy estimate (not
+              implemented). Defaults to 0.
+        - n_sats:
+            type: u8
+            desc: Number of satellites used in solution.
+        - vel_n:
+            type: s32
+            units: mm/s
+            desc: Velocity North coordinate
+        - vel_e:
+            type: s32
+            units: mm/s
+            desc: Velocity East coordinate
+        - vel_d:
+            type: s32
+            units: mm/s
+            desc: Velocity Down coordinate
+        - pdop:
+            type: u16
+            units: 0.01
+            desc: Position Dilution of Precision
+        - flags:
+            type: u8
+            desc: Status flags
+            fields:
+              - 0-2:
+                  desc: Fix mode
+                  values:
+                    - 0x000: Invalid
+                    - 0x001: RTK Float
+                    - 0x010: RTK Fixed
+                    - 0x011: SPP
+                    - 0x100: Reserved
+                    - 0x101: Reserved
+                    - 0x110: Reserved
+                    - 0x111: Reserved
+              - 3-5:
+                  desc: UTC flags
+                  values:
+                    - 0x00: Invalid UTC date
+                    - 0x01: Valid UTC date, defaults used
+                    - 0x10: Valid UTC leap seconds
+              - 7:
+                  desc: RAIM repair flag
+                  values:
+                    - 0x0: No RAIM repair
+                    - 0x1: RAIM repair
```

Comment:
Comments:
- add gps week
- make lat/lon as scaled integer
- use altitude instead of height
- n_sats rename to n_sats_used
- replace vel_x with commonly used SOG and COG
- fix mode flag: value 3 for SPP is rather unexpected, better use:
    0 - no fix
    1 - SPS
    2 - DGPS (reserved)
    3 - RTK Float
    4 - RTK Fixed
- UTC flags should be the same as ones used in UTC message Dennis is preparing
- remove RAIM repair, I see no value for user
- do we ever expect 2D fixes with invalid altitude? May need more individual field validity flags.
- add HDOP
- add data latency